ParkingZone.Domain/Entities/ParkingZone.cs

```diff
@@ -0,0 +1,13 @@
+﻿using ParkingZone.Domain.Commons;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace ParkingZone.Domain.Entities;
+
+public class ParkingZone : Auditable
+{
+    public string Name { get; set; }
+     public string Country { get; set; }
+    public string City { get; set; }
+    public string Region { get; set; }
+    public string Street { get; set; }
```

Comment:
Just add `Address` property instead of these ones.
Also add `DateOfEstablishment` property.